### ISSUE TITLE
Double check: Don't hear solute name when changing solute with solution values not checked

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#163]
I hear hear 
- some drink, Solute Amount, slider
- some drink mix 
- a bunch of drink mix 
but I do not hear the solute name when Solution Values is not checked.

For example, I hear:
- 0.500 moles, Solute Amount, slider
- 0.550 moles
- 0.600 moles

I just want to make sure this is intentional. It might very well be because the solute names are very long.

The design doc, though, has the solute name in:
- 0.500 moles of drink mix
- 0.550 moles of drink mix

**Comment by Matthew-phet** [comment]
Yeah I think it was intentional. Do we want to add the name to the values or keep it as is?

**Comment by terracoda** [comment]
Let's keep it as is, for now, and quickly review again after #160 and #154 are addressed.

**Comment by terracoda** [comment]
I'll mark on hold for now.

**Comment by terracoda** [comment]
I double checked with @Matthew-phet and we decided at some point to not include the name of the solute when Solution Values checkbox is checked.

Closing.
